Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -49,17 +54,61 @@ class SCENE_SURFACE_MESH_ITEM_EXPORT Scene_surface_mesh_item
   Bbox bbox() const Q_DECL_OVERRIDE;
   QString toolTip() const Q_DECL_OVERRIDE;
 
+  // Only needed for Scene_polyhedron_item
+  void setItemIsMulticolor(bool);
+  void update_vertex_indices(){}
+  void update_halfedge_indices(){}
+  void update_facet_indices(){}
+  Vertex_selection_map vertex_selection_map();
+  Face_selection_map face_selection_map();
+
+  std::vector<QColor>& color_vector();
+
+  void set_patch_id(SMesh::Face_index f,int i)const;
+  int patch_id(SMesh::Face_index f)const;
```

Comment:
The code of `Scene_surface_mesh_item.h` is wrong: it should not expose any CGAL API.

- All the typedefs `<foo>_descriptor`,
- all the functions that access or modify properties of elements of the mesh (instead of global properties of the item) 
should be moved to another file, such as the file `Polyhedron_type.h` for polyhedron.

Anyway, I have checked and, for example, the function `Scene_surface_mesh_item::set_patch_id` is never used anywhere in the demo.